libs/eo/consumption-page/shell/src/lib/eo-consumption-page-shell.component.ts

```diff
@@ -56,11 +59,15 @@ import { EoConsumptionStore } from './eo-consumption.store';
           class="watt-space-stack-l"
         ></eo-consumption-line-chart>
         <eo-date-picker
+          class="watt-space-stack-l"
           [onFeatureFlag]="'daterange'"
           *rxLet="appSettingsDates$ as dates"
           [dateRangeInput]="dates"
           (newDates)="setNewAppDates($event)"
         ></eo-date-picker>
+        <eo-resolution-picker
+          [onFeatureFlag]="'resolution'"
```

Comment:
I mean... im not complaining that you made a new feature flag, but the idea was the hide resolution and datepicker behind the datepicker flag 😆 But good! Thank you for using it 😄